Extract MySQL instructions to separate pages (#582)
appsignal/appsignal-docs: source/nodejs/installation/opentelemetry.html.md

const opentelemetry = require("@opentelemetry/sdk-node");
const { NodeTracerProvider } = require('@opentelemetry/sdk-trace-node');

// Import the necessary instrumentation

> // This will be done in the next steps

Reviewer: Since requiring the package is already part of the code example given in each of the instrumentation-specific pages, I would just remove this comment here.

Author: But this way people know where to put it.

// Configure OpenTelemetry to instrument specific libraries
const sdk = new opentelemetry.NodeSDK({

>   // Configure the necessary instrumentation
>   // This will be done in the next steps

Reviewer: ```suggestion
  // Configure the desired instrumentations here, as described in the next steps
  instrumentations: []
```

I think the empty array provides some context as to what you will be doing in the next steps.
